idea/src/META-INF/plugin.xml

```diff
@@ -2539,8 +2539,14 @@
                      groupName="Redundant constructs"
                      enabledByDefault="true"
                      level="WEAK WARNING"
-                     language="kotlin"
-    />
+                     language="kotlin"/>
+
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.RemoveRedundantBackticksInspection"
+                     displayName="Remove redundant backticks"
+                     groupName="Kotlin"
```

Comment:
Please specify groupPath and groupName as in other Kotlin inspections (the groupName should be "Redundant constructs")

Reply:
Fixed